src/api/goods.js

```diff
@@ -6,6 +6,14 @@ export function getAll() {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => (
+  getAll()
+    .then(goods => goods.sort(
+      (prev, next) => prev.name.localeCompare(next.name),
+    ).splice(0, 5))
```

Comment:
Краще не мутувати масив
```suggestion
    ).slice(0, 5))
```